### DISCUSSION TITLE
Samsung S22 Android 13 ( X drip version e13a76e-2022.08.03) Bluetooth drop out issues

### DISCUSSION THREAD

**Discussion by andrewkerr1991** [NightscoutFoundation/xDrip#2525] [Q&A]
Hi There, 

Navid Fo has kindly been supporting me on X drip FB group but has asked me to post on here as we are running out of ideas. 

I am running Xdrip version e1376e-2022.08.03 on a Samsung S22 ( photos attached for reference) , every night I am getting a drop out which is disconnecting X drip and Android APS.  I have followed the suggestions on the guide below and all settings in x drip are set as per G6 guide (bluetooth & other misc options)

https://navid200.github.io/xDrip/docs/Intermittent?fbclid=IwAR1YFO-i5TLiCsqy9fPYSHXXAtTRAs_ZdBFkDZPEkxs08wNvXd9NLRsNTj0

Here is the link to the FB discussion for reference also. 

https://www.facebook.com/groups/xDripG5/posts/8298621840211238/?comment_id=8298775933529162&reply_comment_id=8311128955627193&notif_id=1669842195940451&notif_t=group_comment_mention

I have attached the screenshots of xdrip version and event logs, so hopefully this will help. 

Does anyone have any ideas how I can stop this drop out ? 

Thanks

Andrew 

![Bluetooth settings](https://user-images.githubusercontent.com/59082024/205458721-c5ca70fe-cccd-44e2-aa80-95ccc36fc714.jpg)
![Debug settings 1](https://user-images.githubusercontent.com/59082024/205458722-61027508-a0e8-4660-ae5f-641a7687d95b.jpg)
![Debug settings 2](https://user-images.githubusercontent.com/59082024/205458723-ea32c96e-fee8-46e8-83c4-c1a6b359b624.jpg)
![Debug settings](https://user-images.githubusercontent.com/59082024/205458725-8417d8ce-62fc-4730-a4a5-8a365f71f5f9.jpg)
![Event log 1](https://user-images.githubusercontent.com/59082024/205458726-b5183444-3656-4a78-b985-3d3cb44429cf.jpg)
![event log 2](https://user-images.githubusercontent.com/59082024/205458727-b8fd1b33-93f2-420f-91ed-7d93d7e854ce.jpg)
![event log 3](https://user-images.githubusercontent.com/59082024/205458729-9ea00d61-7fad-46f4-82e3-bc7f6fc259a7.jpg)
![event log 4](https://user-images.githubusercontent.com/59082024/205458731-b530b300-33b6-4c97-adb4-9f1d4e0c1205.jpg)
![event log](https://user-images.githubusercontent.com/59082024/205458733-ecc09dbe-4374-4ad4-9695-0bc1885b3870.jpg)
![other misc options](https://user-images.githubusercontent.com/59082024/205458735-512be8d0-bfb3-4988-a5dc-dca1207db12e.jpg)
![S22](https://user-images.githubusercontent.com/59082024/205458736-9b06fddc-e133-47c8-bac4-ef4f1d9d0f08.jpg)
![Xdrip system status](https://user-images.githubusercontent.com/59082024/205458738-7e84361e-bef3-40c4-be78-3ff8ba5a05db.jpg)

**Comment by Navid200** [comment]
Would you please try this version?  https://github.com/NightscoutFoundation/xDrip/releases/tag/2022.11.27
Just install over what you have.  Please don't uninstall.

**Reply by andrewkerr1991** [reply]
Just an update for any developers here... I have been having continued issues with this drop out. I installed the latest version of Xdrip 5ea552c-2022.12.09 but still had the blue tooth disconnection issues. I turned off the "special pairing workaround" for Samsung and I had no connection issues for over 4 days. Hope this helps. Let me know if you require any information I may have missed.

**Reply by Navid200** [reply]
Would you please include a screenshot?
There is a Samsung workaround setting in Bluetooth settings.
And there is a special pairing workaround, which does not have "Samsung" in the title.

So, it is not clear from your post which one you mean.

**Reply by Navid200** [reply]
Is this the setting you disabled?
![Screenshot_20221216-103602](https://user-images.githubusercontent.com/51497406/208134103-e72afb55-c3c3-4216-a839-508f6acbbc81.png)

**Reply by andrewkerr1991** [reply]
Correct, I was just pasting a screen shot. I put the " special pairing workaround" . Sorry thought that was clear.
